lib/util/player_configuration.js

```diff
@@ -119,6 +119,8 @@ shaka.util.PlayerConfiguration = class {
       startAtSegmentBoundary: false,
       smallGapLimit: 0.5,
       jumpLargeGaps: false,
+      gapThreshold: 0.1, /* seconds */
+      gapJumpingPollInterval: 0.25, /* seconds */
```

Comment:
These comments aren't really needed since the comments on the configuration documentation should be enough.